languagetool-core/src/main/java/org/languagetool/rules/neuralnetwork/Dictionary.java

```diff
@@ -0,0 +1,35 @@
+package org.languagetool.rules.neuralnetwork;
+
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.List;
+
+public class Dictionary extends HashMap<String, Integer> {
+
+  public Dictionary(InputStream filePath) {
+    List<String> rows = ResourceReader.readAllLines(filePath);
+    fromString(rows.get(0));
+  }
+
+  public Dictionary(String dict) {
+    fromString(dict);
+  }
+
+  private void fromString(String maps) {
+    maps = maps.substring(1, maps.length() - 1);
+    for (String entry : maps.split(", ")) {
+      String[] kv = entry.split(": ");
+      put(kv[0].substring(1, kv[0].length() - 1), Integer.parseInt(kv[1]));
+    }
+  }
+
+  Integer safeGet(String key) {
+    if (containsKey(key)) {
+      return get(key);
+    } else {
+      System.out.println(key + " unknown");
```

Comment:
Yeah, can be removed. We don't have a logging system in LT, do we?

Reply:
No, it was once planned, but for now, we assume that we're a library: either everything is fine, or if it's not, we can throw an exception instead of logging that something might be wrong.